### DISCUSSION TITLE
MQTT & unused I/O pins.

### DISCUSSION THREAD

**Discussion by dewy721** [omersiar/esp-rfid#528] [Q&A]
Is there a way to use MQTT to control the unused I/O pins?

The readers we use have an RGB status led, and a piezo buzzer for user feedback. Breakouts with one wire for each.

**Comment by matjack1** [comment]
hey @dewy721 currently you cannot use unused I/O pins unfortunately.

You would like to drive a specific pin with a MQTT call directly? Bypassing all the esp-rfid functionality?

esp-rfid already manages a led and buzzer for feedback, so I'm not sure what you have in mind in detail?

**Comment by dewy721** [comment]
Well, I was imaging connections for i2c lcd output (for MQTT based
messages).

**Comment by dewy721** [comment]
MQTT direct I/O? Sounds useful as well.

**Comment by matjack1** [comment]
OK, that sounds interesting :)

Currently there's nothing to drive this though. If you want to work on this feel free to ask, I can help you to get started!
